src/kernel/src/dev/dmem.rs

```diff
@@ -0,0 +1,16 @@
+use crate::{
+    errno::Errno,
+    fs::{Device, IoCmd},
+    process::VThread,
+};
+use std::sync::Arc;
+
+#[derive(Debug)]
+struct Dmem {}
+
+impl Device for Dmem {
+    #[allow(unused_variables)] // TODO: remove when implementing
+    fn ioctl(self: Arc<Self>, cmd: IoCmd, td: Option<&VThread>) -> Result<(), Box<dyn Errno>> {
+        todo!()
```

Comment:
Why not bringing what we have implemented in the `src/kernel/src/fs/dev/dmem1.rs` here?

Reply:
Good point